### ISSUE TITLE
Error: Style property 'width' is not supported by native animated module

### ISSUE THREAD

**Issue by Madhu02** [pritishvaidya/react-native-password-strength-meter#17]
Hi,

I have tried implementing the your library, accessing it as a Component Usage. 

This is the error i'm getting with your library.

[Mon Oct 05 2020 14:00:34.877]  ERROR    Error: Style property 'width' is not supported by native animated module

This error is located at:
    in AnimatedComponent (at createAnimatedComponent.js:215)
    in ForwardRef(AnimatedComponentWrapper) (at bar-password-strength-display.js:51)
    in RCTView (at View.js:34)
    in View (at bar-password-strength-display.js:50)
    in RCTView (at View.js:34)
    in View (at bar-password-strength-display.js:49)
    in BarPasswordStrengthDisplay (at PasswordScreen.tsx:105)
    in RCTView (at View.js:34)
    in View (at PasswordScreen.tsx:104)
    in RCTView (at View.js:34)
    in View (at PasswordScreen.tsx:90)
    in RCTView (at View.js:34)
    in View (at PasswordScreen.tsx:76)
    in RCTView (at View.js:34)
    in View (at PasswordScreen.tsx:72)
    in RCTScrollContentView (at ScrollView.js:1124)
    in RCTScrollView (at ScrollView.js:1260)
    in ScrollView (at ScrollView.js:1286)
    in ScrollView (at KeyboardAwareHOC.js:517)
    in KeyboardAwareScrollView (at PasswordScreen.tsx:70)
    in RCTSafeAreaView (at SafeAreaView.js:51)
    in ForwardRef(SafeAreaView) (at PasswordScreen.tsx:69)
    in PasswordScreen (at SceneView.tsx:122)
    in StaticContainer
    in StaticContainer (at SceneView.tsx:115)
    in EnsureSingleNavigator (at SceneView.tsx:114)
    in SceneView (at useDescriptors.tsx:150)
    in RCTView (at View.js:34)
    in View (at CardContainer.tsx:221)
    in RCTView (at View.js:34)
    in View (at CardContainer.tsx:220)
    in RCTView (at View.js:34)
    in View (at CardSheet.tsx:33)
    in ForwardRef(CardSheet) (at Card.tsx:563)
    in RCTView (at View.js:34)
    in View (at createAnimatedComponent.js:165)
    in AnimatedComponent (at createAnimatedComponent.js:215)
    in ForwardRef(AnimatedComponentWrapper) (at Card.tsx:545)
    in PanGestureHandler (at GestureHandlerNative.tsx:13)
    in PanGestureHandler (at Card.tsx:539)
    in RCTView (at View.js:34)
    in View (at createAnimatedComponent.js:165)
    in AnimatedComponent (at createAnimatedComponent.js:215)
    in ForwardRef(AnimatedComponentWrapper) (at Card.tsx:535)
    in RCTView (at View.js:34)
    in View (at Card.tsx:529)
    in Card (at CardContainer.tsx:189)
    in CardContainer (at CardStack.tsx:558)
    in RCTView (at View.js:34)
    in View (at Screens.tsx:69)
    in MaybeScreen (at CardStack.tsx:551)
    in RCTView (at View.js:34)
    in View (at Screens.tsx:48)
    in MaybeScreenContainer (at CardStack.tsx:461)
    in CardStack (at StackView.tsx:458)
    in KeyboardManager (at StackView.tsx:456)
    in RNCSafeAreaProvider (at SafeAreaContext.tsx:74)
    in SafeAreaProvider (at SafeAreaProviderCompat.tsx:42)
    in SafeAreaProviderCompat (at StackView.tsx:453)
    in RCTView (at View.js:34)
    in View (at StackView.tsx:452)
    in StackView (at createStackNavigator.tsx:85)
    in StackNavigator (at App.tsx:26)
    in EnsureSingleNavigator (at BaseNavigationContainer.tsx:390)
    in ForwardRef(BaseNavigationContainer) (at NavigationContainer.tsx:91)
    in ThemeProvider (at NavigationContainer.tsx:90)
    in ForwardRef(NavigationContainer) (at App.tsx:25)
    in App (at App.tsx:99)
    in Provider (at App.tsx:98)
    in _default (at renderApplication.js:45)
    in RCTView (at View.js:34)
    in View (at AppContainer.js:106)
    in RCTView (at View.js:34)
    in View (at AppContainer.js:132)
    in AppContainer (at renderApplication.js:39)

**Comment by Madhu02** [comment]
Seems like the there was modification in the library after raising this issue, how do i pull the latest from the library ? I have installed the library but it is throwing me warnings as below
```
$ npm i react-native-password-strength-meter --save
npm WARN react-native-fs@2.16.6 requires a peer of react-native@^0.59.5 but none is installed. You must install peer dependencies yourself.
npm WARN react-native-fs@2.16.6 requires a peer of react-native-windows@^0.57.2 but none is installed. You must install peer dependencies yourself.
npm WARN react-native-zip-archive@5.0.6 requires a peer of react-native@^0.60.0 but none is installed. You must install peer dependencies yourself.
npm WARN react-native-password-strength-meter@0.0.4 requires a peer of react-native@^0.57.1 but none is installed. You must install peer dependencies yourself.
npm WARN react-native-speedometer@1.0.5 requires a peer of react-native@^0.57.1 but none is installed. You must install peer dependencies yourself.
```

Are these peer dependencies should be installed ?

**Comment by expertcoder17** [comment]
Error: Style property 'width' is not supported by native animated module #17

Any update in same issue.Need urgent for this.

**Comment by Madhu02** [comment]
@expertcoder17  Nope, no response for my last comment. So i have implemented on my own using https://www.npmjs.com/package/react-native-zxcvbn.

**Comment by ab-sami** [comment]
Any update on this issue?

**Comment by pritishvaidya** [comment]
It should be fixed now
